server/src/processing/NumberCorrectionModule/NumberCorrectionModule.ts

```diff
@@ -72,12 +60,12 @@ export class NumberCorrectionModule extends Module<Options> {
 	}
 
 	public main(doc: Document): Document {
-		const options = Object.assign(defaultOptions, this.options);
+		// const options = Object.assign(defaultOptions, this.options);
```

Comment:
Please check this!
I removed the Object.assign() as that kind of merge is no longer valid, and just used the `this.options` built in Module constructor

Reply:
👍